src/signals/incident/components/form/MapSelectors/Asset/Selector/Selector.test.tsx

```diff
@@ -183,9 +183,11 @@ describe('signals/incident/components/form/AssetSelect/Selector', () => {
       clientY: 10,
     })
 
-    expect(fetchLocation).toHaveBeenCalledWith(
-      expect.not.objectContaining(coordinates)
-    )
+    setTimeout(() => {
```

Comment:
Jest has the ability to use mocked timers. That gives a lot more control over test environments and will not delay the test execution. See https://jestjs.io/docs/timer-mocks

Reply:
✅